.github/workflows/tests.yaml

```diff
@@ -182,71 +39,32 @@ jobs:
             const fs = require('fs');
             const result = JSON.parse(fs.readFileSync('./test-result.json')).testenvs;
 
-            let lint_result = result.lint.test;
-            let lint_success = true;
-            let lint_output = '';
-            for (let lint_test_result of lint_result) {
-              if (lint_test_result.retcode != 0) {
-                lint_success = false;
-              }
-              if (lint_test_result.output) {
-                lint_output += lint_test_result.output;
-              }
-            }
-            let static_result = result.static.test;
-            let static_output = static_result[0].output;
+            let int_result = result.test.test;
+            let int_success = int_result[0].retcode == 0;
+            let int_output = int_result[0].output;
```

Comment:
I think you meant `lint`, as in `lint_result` :D

Reply:
as mentioned before, am not the original author, but yeah will change that.